ext/session/session.c

```diff
@@ -1531,36 +1542,50 @@ PHPAPI void php_session_start(void) /* {{{ */
 		PS(id) = NULL;
```

Comment:
@yohgaki Thank you for clarifying, the session module does seem a mess at some parts and I was just trying to understand it a bit better. I will let you know if I spot anything obvious.